### PR TITLE
Remove reset, clean up repo

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: packages/css/src/link/link.scss [hunk 1]
```diff
@@ -5,14 +5,22 @@
 
 @import "../../utils/calculate-fluid-style";
 
+@mixin reset {
+  background-color: transparent;
```
Comment (VincentSmedinga) [inline packages/css/src/link/link.scss:9]: Which style does this override?

**Review by VincentSmedinga — APPROVED** [review]

File: storybook/storybook-css/config/preview.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* Use border-box sizing by default for all elements */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (alimpens) [reply storybook/storybook-css/config/preview.css]: Good catch. I thought the Storybook docs needed this, but on second thought, this isn't really necessary. I removed it

File: packages/tokens/src/brand/amsterdam/typography.tokens.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "amsterdam": {
+    "typography": {
+      "font-family": {
+        "value": "'Amsterdam Sans', Arial, sans-serif"
+      },
+      "font-weight": {
+        "normal": {
+          "value": 400
+        },
+        "bold": {
```
[the remark this answers is not in the document]
Comment (alimpens) [reply packages/tokens/src/brand/amsterdam/typography.tokens.json:11]: The designs are a little messy here. They say we use both bold and extra bold. In [the linked Figma file](https://www.figma.com/file/Re56SrOKad96jvbLZC0YYF/Design-bibliotheek?node-id=150%3A1390&t=akTC88gmSCPemRQG-0) `font-weight: 700` is only used in combination with Avenir? In [the new Figma file](https://www.figma.com/file/ORa7CBIooPgZj6HsEPBxNR/Design-bibliotheek-(gepubliceerd)?node-id=150%3A1326&t=UTIZz82OjhiI87o9-0) only font weights 400 and 800 are used. I assumed the newer Figma file is correct, but we should check

File: packages/css/src/link/link.scss (hunk 1 above)
Comment (alimpens) [reply packages/css/src/link/link.scss:9]: [It's this part of normalize.css](https://github.com/necolas/normalize.css/blob/master/normalize.css#L72-L78)

File: packages/tokens/src/brand/amsterdam/typography.tokens.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply packages/tokens/src/brand/amsterdam/typography.tokens.json:11]: It does seems that only regular and extra-bold are being used, and I agree to use ‘bold’ as a more readable API for the latter.

File: packages/css/src/link/link.scss (hunk 1 above)
Comment (VincentSmedinga) [reply packages/css/src/link/link.scss:9]: I don’t think we’ll be supporting IE 10, let’s remove this.

**Pull request merged**